vcd/sample_vcd_test_config.json

```diff
@@ -90,6 +90,44 @@
     "uploadPieceSize": 5,
     "uploadProgress": true
   },
+  "testEnvBuild": {
+    "//": "This section is optional for running tests. It's only needed to build the test environment in a new vCD",
+
+    "// ": "Entities that will be created in an empty vCD use names defined above",
+
+    "// ": "vcd.org                    -> organization name",
+    "// ": "vcd.vdc                    -> VDC name",
+    "// ": "vcd.catalog.name           -> catalog name",
+    "// ": "vcd.catalog.catalogItem    -> catalog item name",
+    "// ": "networking.edgeGateway     -> edge gateway name",
+    "// ": "networking.externalNetwork -> external network name",
+
+    "//": "Values used to build a new environment. Build will fail if the next 5 ones are empty",
+    "gateway": "10.15.21.1",
+    "netmask": "255.255.224.0",
+    "externalNetworkStartIp": "10.15.21.10",
+    "externalNetworkEndIp": "10.15.21.11",
+    "dns1": "10.15.21.101",
+    "dns2": "8.8.4.4",
+
+    "//": "external network may be different from the one defined for testing",
+    "externalNetworkPortGroup": "network_name_from_vcenter",
+    "externalNetworkPortGroupType": "NETWORK",
+
+    "//": "leave the storage profile empty to skip its usage",
+    "storageProfile2": "Development",
+
+    "//": "catalog and media paths can be different from the ones used for testing",
+    "ovaPath": "/home/username/vm/photon-hw11-3.0-26156e2.ova",
+    "media_path":  "/home/username/vm/test.iso",
```

Comment:
This ISO looks like the one shipped with our code. Can we point this path to the included ISO, so it's one variable less to worry about by default?

Reply:
> This ISO looks like the one shipped with our code. Can we point this path to the included ISO, so it's one variable less to worry about by default?

This item has been here for months and its usage is unrelated to this PR. Let's leave its simplification for when we know enough to refactor the whole confifuration file.